config: fix schema:get/schema:set() changes passed path (#10886)
tarantool/tarantool: changelogs/unreleased/gh-10855-config-get-changes-path.md

## bugfix/config

> * `config:get()` no longer changes the passed argument if it's been

Reviewer: ```suggestion
* `config:get()` no longer changes the passed argument if it is
```

Author: Applied. Thank you

>   passed as a table (gh-10855).

Reviewer 2: `<schema object>:get()` and `<schema object>:set()` are fixed as well. Shold we reflect it in the changelog entry?

Author: Applied. Thank you.

Author: Ah, nvm, I've to add the tests on `:set()` too.

Author: Applied X2. Thank you for suggestion.

Author: I've mentioned it in the changelog. Thank you.
